hi-ml-histopathology/src/SSL/configs/CRCK_SimCLRContainer.py

```diff
@@ -0,0 +1,60 @@
+from enum import Enum
+from pathlib import Path
+from typing import Any, Optional
+import sys
+
+from SSL.lightning_containers.ssl_container import EncoderName, SSLContainer, SSLDatasetName
+from SSL.utils import SSLTrainingType
+from histopathology.datasets.tcga_crck_tiles_dataset import TcgaCrck_TilesDatasetWithReturnIndex
+from SSL.configs.HistoSimCLRContainer import HistoSSLContainer
+
+current_file = Path(__file__)
+print(f"Running container from {current_file}")
+print(f"Sys path container level {sys.path}")
+
+local_mode = False
+path_local_data: Optional[Path]
+if local_mode:
+    is_debug_model = True
+    num_workers = 0
+    max_epochs = 2
+else:
+    is_debug_model = False
+    num_workers = 12
+    max_epochs = 200
```

Comment:
I think this block (line 15-24) can be changed in the same way as we did for the deepmil configs. Happy to discuss if there are questions